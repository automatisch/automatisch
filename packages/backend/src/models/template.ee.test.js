import { describe, it, expect } from 'vitest';
import Template from './template.ee.js';

describe('Template model', () => {
  it('tableName should return correct name', () => {
    expect(Template.tableName).toBe('templates');
  });

  it('jsonSchema should have correct validations', () => {
    expect(Template.jsonSchema).toMatchSnapshot();
  });
});
