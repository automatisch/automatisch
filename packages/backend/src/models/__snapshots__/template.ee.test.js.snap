// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Template model > jsonSchema should have correct validations 1`] = `
{
  "properties": {
    "createdAt": {
      "type": "string",
    },
    "flowData": {
      "type": "object",
    },
    "id": {
      "format": "uuid",
      "type": "string",
    },
    "name": {
      "minLength": 1,
      "type": "string",
    },
    "updatedAt": {
      "type": "string",
    },
  },
  "required": [
    "name",
  ],
  "type": "object",
}
`;
